src/components/Auth/Signup.tsx

```diff
@@ -41,7 +41,7 @@ export const Signup: React.FC<Props> = ({ onClose, onOpenLogin }) => {
   const onSubmit = async () => {
     const formData = getValues();
 
-    dispatch(signupUser(formData))
+    dispatch(signupUser({ ...formData, wishList: [] }))
```

Comment:
I think it can be `wishList: [] ` default behaviour on BE

Reply:
@VYaniv, done